docs/docs/spark-procedures.md

```diff
@@ -823,6 +823,8 @@ second snapshot deleted 1 record.
 |2	| Bob	   |INSERT	|0	|5390529835796506035|
 |1	| Alice  |DELETE	|1	|8764748981452218370|
 
+#### Net Changes
```

Comment:
This is an item under the subtitle `example`, it should not be the same level as `example`. Can we change it to one level down, like `##### Net Changes`?

Reply:
I think `Net Changes` is a feature like `Pre/Post Update Images` so they should be on the same level.

Reply:
I'm OK to bring it up to the same level as `Pre/Post Update Images`. By doing that, we will need to give a brief introduction before jumping into an example.  Something like this:
```
The procedure can remove intermediate changes across multiple snapshots, and only outputs the net changes. Here is an example to create a changelog view that computes net changes. 
```